examples/pass-parameters-to-tests/README.md

```diff
@@ -0,0 +1,23 @@
+# Pass Parameters to Tests
+
+**Test Code**: [index.js](index.js)
+
+This example demonstrates how to pass parameters to TestCafe tests. The parameter values can be obtained from the following sources:
+
+* a JSON configuration file
+* a JavaScript object
+* an environment variable
+
+Before test execution, the parameters are obtained. Two of them are loaded from `.json` and `.js` files with an [import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) statement, the third is set as an environment variable with the [process.env](https://nodejs.org/api/process.html#process_process_env) property.
+
+For each test, a `test.page` method call with an appropriate parameter sets the initial page. A `clientFunction` obtains the [File URL](https://nodejs.org/api/url.html#url_url_pathtofileurl_path) of the current page. The `t.expect.eql` method compares this value to the full path to the file that the configuration file or the environment variable specified.
+
```

Comment:
The `t.expect.eql` method compares this value to the full path calculated based on parameter.

Reply:
> The `t.expect.eql` method compares this value to the full path calculated based on the specified parameter.